ci/pipeline.yml

```diff
@@ -13,6 +13,7 @@ steps:
       PLATFORM: '{{name}}'
     artifact_paths:
       - "tmp/build/**/*"
+      - "bazel-testlogs/**/*.log"
```

Comment:
Actually, let's do `"bazel-testlogs/**/test.log"`, since I don't think anyone needs to look at `test.xml.log` files.